proto/mantrachain/sanction/v1/tx.proto

```
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package mantrachain.sanction.v1;
+
+import "cosmos/msg/v1/msg.proto";
```

Buf (1.47.2)

import "cosmos/msg/v1/msg.proto": file does not exist

(COMPILE)